### PR TITLE
Fix for release build.

### PR DESCRIPTION
**Pull request by noobs2ninjas**

### CODE REVIEW

**Comment by noobs2ninjas** [comment]
What a pain. We need to rewrite our testing sometime soon. Because this uses carthage submodules you have to delegate all carthage anything. Actually commit it. Then run your carthage update. Waste of time.

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,5 +1,4 @@
 github "BoltsFramework/Bolts-ObjC" "1.9.0"
 github "BoltsFramework/Bolts-Swift" "e7811181243e9dfa9462ec3048cd5957519fa26f"
-github "ParsePlatform/Parse-SDK-iOS-OSX" "1.17.1"
+github "ParsePlatform/Parse-SDK-iOS-OSX" "1.16.0"
```
Comment (TomWFox) [inline Cartfile.resolved:3]: Why not revert to 1.17.0?

Comment (noobs2ninjas) [reply Cartfile.resolved:3]: Wasnt there an issue that affected queries with 1.17.0 or did that also exist in 1.16.0.

Comment (TomWFox) [reply Cartfile.resolved:3]: I think you’re right. If you think 1.16.0 is the right move then that’s fine

Comment (noobs2ninjas) [reply Cartfile.resolved:3]: I think so. Just need your approval.

**Review by TomWFox — APPROVED** [review]

**Pull request merged**